plugins/example_plugin.bash

```diff
@@ -79,7 +79,11 @@ restore_data() {
   timestamp_day_dir=${timestamp_dir%??????}
   if [ -e "/tmp/GPBACKUP_PLUGIN_LOG_TO_STDERR" ] ; then
     echo 'Some plugin warning' >&2
-  elif [ -e "/tmp/GPBACKUP_PLUGIN_DIE" ] ; then
+  elif [ -e "/tmp/GPBACKUP_PLUGIN_DIE" -a "$GPBACKUP_PLUGIN_DIE_ON_OID" = "" ] ; then
+    exit 1
+  elif [[ -e "/tmp/GPBACKUP_PLUGIN_DIE" && "$filename" == *"$GPBACKUP_PLUGIN_DIE_ON_OID"* ]] ; then
```

Comment:
Maybe use a slightly more specific pattern?
```suggestion
  elif [[ -e "/tmp/GPBACKUP_PLUGIN_DIE" && "$filename" == *"$GPBACKUP_PLUGIN_DIE_ON_OID.gz" ]] ; then
```

Reply:
> This works fine, but maybe use a slightly more specific pattern?

One could try to use the same mechanism in other tests somewhen in future. By this change we will limit available options to gzip, while there is also  zstd supported. Do you really want to limit supported filenames to gz only?

Reply:
I thought so because some oids may collide with the restore key and trigger on every file. In this case, 16392 is unlikely to collide, but other oids could. 

> One could try to use the same mechanism in other tests somewhen in future.

That's a good reason. I think it's up to you.

Reply:
> That's a good reason. I think it's up to you.

Then let's leave it as is for now